aptos-move/move-examples/defi/sources/locked_coins.move

```diff
@@ -38,35 +37,38 @@ module defi::locked_coins {
         withdrawal_address: address,
         // Number of locks that have not yet been claimed.
         total_locks: u64,
-
-        cancel_lockup_events: EventHandle<CancelLockupEvent>,
-        claim_events: EventHandle<ClaimEvent>,
-        update_lockup_events: EventHandle<UpdateLockupEvent>,
-        update_withdrawal_address_events: EventHandle<UpdateWithdrawalAddressEvent>,
     }
 
+    #[event]
     /// Event emitted when a lock is canceled.
-    struct CancelLockupEvent has drop, store {
+    struct CancelLockup has drop, store {
```

Comment:
Is it going to be the convention to drop the suffix "Event"?

Reply:
My current pattern for aptos-framework is:
1. If the current event struct doesn't need add new field to migrate to v2, I will just add `#[event]` attribute directly to that struct to spare the hassle of creating a new struct.
2. otherwise, create a new struct without `Event` suffix since attribute can tell whether it is an event or not and you cannot share the same event name in the same module.

Does it make sense?

Reply:
Make sense!
What if the current event v1 struct (for example AbcEvent) needs to add a new field? It will become AbcEventV2 or AbcV2?

Reply:
there is no way for them to add field. I think any new v2 does not need `Event` string in their name.